build.js

```diff
@@ -21,19 +21,19 @@ var fs              = require('fs')
 					'statics',
 
 					'autoFiller',
-					'cellTypeHandlers',
+					'CellTypeHandlers',
```

Comment:
do you want me to move all the files with Pascal Scale down below and all the files with camelCase up here?